app/models/protocol_drug.rb

```diff
@@ -0,0 +1,11 @@
+class ProtocolDrug < ApplicationRecord
+  belongs_to :protocol
+  before_create :assign_id
+
+  validates :name, presence: true
+  validates :dosage, presence: true
+
+  def assign_id
+    self.id = SecureRandom.uuid
```

Comment:
We should discuss this. I think we should switch all of these records to create UUIDs as ID by default, or move UUID to a separate field and use bigint IDs. As it is, this is messy; I ran into the same issue with Facility.

Reply:
Creating UUIDs by default in the database sounds like the way to go. We'll do this for all entities that are created via the admin interface.

Reply:
We'll do this after the facilities PR is merged in, so that we don't conflict with your changes of adding pgcrypto or another extension.